docs/source/using/addons.adoc

```diff
@@ -161,6 +161,23 @@ $ minishift addons enable anyuid
 $ minishift addons disable anyuid
 ----
 
+[[apply-addons]]
+== Applying add-ons
+
+Add-ons can be explicitly executed with the link:../command-ref/minishift_addons_apply{outfilesuffix}[`minishift addons apply`] command.
```

Comment:
Agreed, but I look at this as a continuation from the implicit use of Add-ons and lets the user know that apart from the implicit way you also have this option. Anyways it is subjective and we can live without using also.